canvas_modules/common-canvas/__tests__/toolbar/toolbar-test.js

```diff
@@ -69,6 +69,44 @@ describe("Toolbar renders correctly", () => {
 		expect(canvasToolbar.find(".toolbar-spacer")).to.have.length(0);
 	});
 
+	it("should render a Toolbar with medium size buttons", () => {
```

Comment:
This test title doesn't match what the test is doing. Rather than putting tests for both sizes in one test could you split this out into two smaller tests and then set the title appropriately for each test please?